src/perl/Permabit/FileSystem.pm

```diff
@@ -133,7 +133,20 @@ sub fsck {
   my ($self) = assertNumArgs(1, @_);
   my $machine = $self->{machine};
   my $devPath = $self->{device}->getSymbolicPath();
-  $machine->runSystemCmd("sudo fsck -t $self->{fsType} -a $devPath");
+  my $fsckCommand = "sudo fsck -t $self->{fsType} -a $devPath";
+  $log->debug("$machine->{hostname}: $fsckCommand");
```

Comment:
You basically just unrolled runSystemCommand, didn't you? That's probably fine for now but in the long run it might make sense to add an extra parameter or an extra method to allow error codes besides 0.

Reply:
Yes. We've got an annoying random assortment of logging vs not logging and allowing nonzero exit status vs blowing up on failure. A variant to log the command and return the exit status would be helpful.